activerecord-trilogy-adapter.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   }
 
   spec.add_dependency "trilogy", ">= 2.1.1"
-  spec.add_dependency "activerecord", "~> 7.1.a"
+  spec.add_dependency "activerecord", ">= 5.0"
```

Comment:
shouldn't this be
```suggestion
  spec.add_dependency "activerecord", ">= 7.0"
```

Reply:
Ah yes -- I originally had compatibility patches in place to go all the way back to Rails 5.1, but now targeting just Rails 7.0.